modules/biobank/php/analysis.class.inc

```diff
@@ -0,0 +1,84 @@
+<?php declare(strict_types=1);
+/**
+ * Specimen Analysis Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Specimen Analysis instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Entity
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\biobank;
+
+/**
+ * Specimen Analysis Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Specimen Analaysis instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Entity
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+class Analysis extends Process
+{
+    /**
+     * Persistent Instance variables.
+     *
+     * These variables are directly mapped to the fields of the database schema
+     * that describe a specimen entity.
+     *
+     * @access private
+     */
+    private $methodId; //int
```

Comment:
the standard way to annotate property types (at least until typed properties are added in the next version of PHP) is with a `@var` annotation, using that instead of `//int` would allow phan and similar tools to better detect bugs

Reply:
Great! Done.